hyper/http11/connection.py

```diff
@@ -166,6 +178,10 @@ def get_response(self):
 
         self._sock.advance_buffer(response.consumed)
 
+        if(response.status == 101 and 
+           b'upgrade' in headers['connection'] and bytes(H2C_PROTOCOL, 'utf-8') in headers['upgrade']):
```

Comment:
The `bytes()` call here doesn't work on Python 2, so it might be better to replace it with a `.decode('utf-8')` call.